### ISSUE TITLE
Django 1.8, "sqldiff -a" fails with TypeError

### ISSUE THREAD

**Issue by hobbsd** [django-extensions/django-extensions#652]
Using version 1.5.2 on Win7. Here's the end of the traceback:

    File "c:\Users\xxxx\py_envs\xxxx\lib\site-packages\django_extensions\management\commands\sqldiff.py", line 134, in <lambda>
      SQL_TABLE_MISSING_IN_MODEL = lambda self, style, qn, args: style.NOTICE('--Model missing for table: %s' % args[0])
    TypeError: not all arguments converted during string formatting

**Comment by hobbsd** [comment]
The change in version 1.5.3 has eliminated the `TypeError`, but now the tables and models seem to not be associating with each other -- `sqldiff -a` says that every model is missing its table and every table is missing its model. For example:
`-- Model missing for table: auth_group`
...
`-- Model: Group`
`-- Table missing: auth_group`

**Comment by codisto** [comment]
under linux the same issue, Django 1.8.1, django-extensions 1.5.3 
--Model: XXX
-- Table missing: XXX

**Comment by trbs** [comment]
commit dc1a190627df97f7ca9eb5ec06a00261d10defb4 should fix this, there will be a new release 1.5.4 shortly for this fix.

**Comment by hobbsd** [comment]
Version 1.5.4 crashes with an AttributeError:
```
    File "c:\Users\...\lib\site-packages\django_extensions\management\commands\sqldiff.py", line 135, in <lambda>
    SQL_TABLE_MISSING_IN_MODEL = lambda self, style, qn, args: style.NOTICE('--
Model missing for table: %s' % args[0].name)
AttributeError: 'str' object has no attribute 'name'
```

**Comment by trbs** [comment]
Sorry for that. Fixed and pushed 1.5.5

**Comment by hobbsd** [comment]
Thanks -- that was fast!
